tensorbay/opendataset/VGGFace2/loader.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+# pylint: disable=invalid-name
+
+"""Dataloaders of VGGFace2 dataset."""
+
+import csv
+import os
+from collections import defaultdict
+from itertools import islice
+from typing import DefaultDict, Dict, Tuple
+
+from tensorbay.dataset import Data, Dataset
+from tensorbay.label import Classification, LabeledBox2D, LabeledKeypoints2D
+from tensorbay.label.basic import AttributeType
+from tensorbay.opendataset._utility import glob
+from tensorbay.utility import chunked
+
+DATASET_NAME = "VGGFace2"
+_SEGMENT_NAMES = ("train", "test")
+
+
+def VGGFace2(path: str) -> Dataset:
+    """`Visual Geometry Group Face 2 <http://www.robots.ox.ac.uk/~vgg/data/vgg_face/>`_ dataset.
+
+    The file structure should be like::
+
+        <path>
+            test_list.txt
+            train_list.txt
+            label/
+                identity_meta.csv
+                loose_bb_test.csv
+                loose_bb_train.csv
+                loose_landmark_test.csv
+                loose_landmark_train.csv
+                attributes/
+                    01-Male.txt
+                    02-Black_Hair.txt
+                    ...
+            test/
+                n000001/
+                    0001_01.jpg
+                    0002_01.jpg
+                    ...
+                n000009/
+                ...
+            train/
+                n000002/
+                    0001_01.jpg
+                    ...
+                n000003/
+                ...
+
+    Arguments:
+        path: The root directory of the dataset.
+
+    Returns:
+        Loaded :class:`~tensorbay.dataset.dataset.Dataset` instance.
+
+    """
+    root_path = os.path.abspath(os.path.expanduser(path))
+    label_path = os.path.join(root_path, "label")
+    dataset = Dataset(DATASET_NAME)
+    dataset.load_catalog(os.path.join(os.path.dirname(__file__), "catalog.json"))
+
+    all_classifications = _get_classifications(
+        label_path, dataset.catalog.classification.attributes.keys()
+    )
+
+    classification_subcatalog = dataset.catalog.classification
+    for item in all_classifications.values():
```

Comment:
in a dict, item is used to represent the combination of key and value, check the dict method `dict.items`.
Here I thick the item is not a good variable name, maybe `value` or `classification` will be better.